hw/ip/edn/data/edn.hjson

```diff
@@ -0,0 +1,264 @@
+                '''
+        }
+      ]
+    },
+    { 
+      name: "SW_CMD_STS",
+      desc: "EDS command status register",
+      swaccess: "rw",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "0",
+          name: "CMD_RDY",
+          desc: '''This bit indicates when the command interface is ready to accept commands.
+                '''
+        }
+        { bits: "1",
+          name: "CMD_ACK",
+          desc: '''This bit is set when the command request has completed. This bit must
+                   be cleared before a new request is made.
+                '''
+        }
+        { bits: "4",
+          name: "CMD_STS",
+          desc: ''' 
+                This one bit field is the status code returned with the application command ack.
+                0b0: Request completed successfully
+                0b1: Request completed with an error
+                '''
+        }
+      ]
+    },
+    {
+      name: "RESEED_CMD",
+      desc: "EDN csrng reseed command register",
+      swaccess: "wo",
+      hwaccess: "hro",
+      hwext: "true",
+      hwqe: "true",
+      regwen: "REGEN",
+      fields: [
+        { bits: "31:0",
+          name: "RESEED_CMD",
+          desc: '''
+                Writing this register will fill a FIFO with up to 13 command words (32b words).
+                This FIFO will be used to automatically send out a reseed command to the CSRNG 
+                appl interface when in AUTO_REQ_MODE. This command will be sent only after 
+                the MAX_NUM_REQS_BETWEEN_RESEEDS counter value has reached zero.
+                '''
+        }
+      ]
+    },
+    {
+      name: "GENERATE_CMD",
+      desc: "EDN csrng generate command register",
+      swaccess: "wo",
+      hwaccess: "hro",
+      hwext: "true",
+      hwqe: "true",
+      regwen: "REGEN",
+      fields: [
+        { bits: "31:0",
+          name: "GENERATE_CMD",
+          desc: '''
+                Writing this register will fill a FIFO with up to 13 command words (32b words).
+                This FIFO will be used to automatically send out a generate command to the CSRNG 
+                appl interface when in AUTO_REQ_MODE. This command will be sent only after 
+                receiving a command ack from the previous command.
+                '''
+        }
+      ]
+    },
+    { name: "MAX_NUM_REQS_BETWEEN_RESEEDS",
+      desc: "EDN maximum number of requests between reseeds register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      hwqe: "true",
+      fields: [
+        {
+            bits: "31:0",
+            name: "MAX_NUM_REQS_BETWEEN_RESEEDS",
+            desc: '''
+                  Setting this field will set the number of generate requests that can be made 
+                  to CSRNG before a reseed request is made. This value only has meaning when in
+                  AUTO_REQ_MODE. A value of 2^48 is allowed, but this register only supports 
```

Comment:
I don't completely understand what this distinction is between 2^48 and 2^32 ?   Clearly the register is 32 bits wide, but is the counter/backing storage register actually 48 bits wide?  How would you got about writing say, 2^35 into that register?

Reply:
2^48 is the NIST limit, but we will do reseeds before then.  Will be clarified in upcoming revsion.